[hotfix] [docs] Refer to Running in an IDE section early on (#19009)
apache/flink: docs/content/docs/try-flink/datastream.md

Maven will create a folder named `frauddetection` that contains a project with all the dependencies to complete this tutorial.
After importing the project into your editor, you can find a file `FraudDetectionJob.java` (or `FraudDetectionJob.scala`) with the following code which you can run directly inside your IDE.
Try setting break points through out the data stream and run the code in DEBUG mode to get a feeling for how everything works.

> If you want to run directly in you IDE, please read the section [Running in an IDE](#running-in-an-ide).

Reviewer: ```suggestion
Please read the section [Running in an IDE](#running-in-an-ide) for all details on how to run Flink applications in your IDE.
```

> Having trouble running directly inside your IDE? Please read the section [Running in an IDE](#running-in-an-ide).

Reviewer 2: I don't like the "Having trouble" wording. Maybe just "If you wanna run directly in you IDE, please read the section..."

Author: Sure.

Running the project in an IDE may result in a `java.lang.NoClassDefFoundError` exception. This is probably because you do not have all required Flink dependencies implicitly loaded into the classpath.

> * IntelliJ IDEA: Go to Run > Edit Configurations > Modify options > Select `include dependencies with "Provided" scope`. This run configuration will now include all required classes to run the application from within the IDE.

Author: I guess github gets confused. There is no change.
